packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -515,8 +526,40 @@ const CommentBoard = ( { thread, parent, isExpanded, onEdit, onDelete } ) => {
 					) }
 				/>
 			) : (
-				<RawHTML className="editor-collab-sidebar-panel__user-comment">
-					{ thread?.content?.rendered }
+				<RawHTML
+					className={ clsx(
+						'editor-collab-sidebar-panel__user-comment',
+						{
+							'editor-collab-sidebar-panel__resolution-text':
+								isResolutionComment,
+						}
+					) }
+				>
+					{ isResolutionComment
+						? ( () => {
+								const actionText =
+									thread.meta._wp_block_comment_status ===
+									'resolved'
+										? __( 'Marked as resolved' )
+										: __( 'Reopened' );
+								const content = thread?.content?.raw;
+
+								if (
+									content &&
+									typeof content === 'string' &&
+									content.trim() !== ''
+								) {
+									return sprintf(
+										// translators: %1$s: action, %2$s: content.
```

Comment:
```suggestion
										// translators: %1$s: action label ("Marked as resolved" or "Reopened"); %2$s: comment text.
```

The string is so short that it may be hard to tell what the action and content are. Let's make the comment a bit more specific.